README.md

```diff
@@ -83,6 +83,12 @@ The exporter accepts the following command line flags:
 # TYPE flashblade_perf_writes_per_sec gauge
 ```
 
+## Authors
+Prometheus FlashBlade Exporter has been under development at Man Group since 2019 and welcomes contributions.
```

Comment:
Let's just make this "under development since 2019".

Reply:
Fixed!